### PR TITLE
Update user store properties

### PR DESCRIPTION
**Pull request by sherenewso2**
## Purpose
- Remove `SCIMEnabled` property
- Add `UserIdSearchFilter` property
- Add `UserIDAttribute` property

### CODE REVIEW

File: en/docs/setup/configuring-a-read-write-active-directory-user-store.md [hunk 1]
```diff
@@ -141,13 +141,20 @@ Default: identityPerson( Is a custom object class defined in WSO2 Identity Serve
...
 <br />
 Note: email address is considered as a special case in WSO2 products, if you want to set the email address as username, see <a href="../../learn/using-email-address-as-the-username">Using email address as the username</a></p>
 <br/>
 sample values: sAMAccountName</td>
 </tr>
+<tr class="odd">
+<td>UserIDAttribute</td>
+<td>user_id_attribute</td>
+<td>User ID Attribute</td>
+<td><p>The attribute used for uniquely identifying a user entry. The value of the attribute is considered as the user ID.</p>
```
Comment (ashensw) [inline en/docs/setup/configuring-a-read-write-active-directory-user-store.md]: Let's modify this as `The value of the attribute is considered as the unique user ID`

File: en/docs/setup/configuring-a-read-write-ldap-user-store.md [hunk 2]
```diff
@@ -133,11 +133,18 @@ Default: identityPerson( Is a custom object class defined in WSO2 Identity Serve
...
+<td><p>A uniquely identifying attribute that represents the username of the user. Users can be authenticated using their email address, UID, etc. The value of the attribute is considered as the username.</p>
 <p>Default: uid<br />
 <br />
 Note: email address is considered as a special case in WSO2 products, if you want to set the email address as username, see <a href="../../learn/using-email-address-as-the-username">Using email address as the username</a></p></td>
 </tr>
+<tr class="odd">
+<td>UserIDAttribute</td>
+<td>user_id_attribute</td>
+<td>User ID Attribute</td>
+<td><p>The attribute used for uniquely identifying a user entry. The value of the attribute is considered as the user ID.</p>
```
Comment (ashensw) [inline en/docs/setup/configuring-a-read-write-ldap-user-store.md]: Let's modify this as `The value of the attribute is considered as the unique user ID`

File: en/docs/setup/configuring-a-read-only-ldap-user-store.md [hunk 3]
```diff
@@ -133,11 +133,18 @@ Default: identityPerson( Is a custom object class defined in WSO2 Identity Serve
...
+<td><p>A uniquely identifying attribute that represents the username of the user. Users can be authenticated using their email address, UID, etc. The value of the attribute is considered as the username.</p>
 <p>Default: uid<br />
 <br />
 Note: email address is considered as a special case in WSO2 products, if you want to set the email address as username, see <a href="../../learn/using-email-address-as-the-username">Using email address as the username</a></p></td>
 </tr>
+<tr class="odd">
+<td>UserIDAttribute</td>
+<td>user_id_attribute</td>
+<td>User ID Attribute</td>
+<td><p>The attribute used for uniquely identifying a user entry. The value of the attribute is considered as the user ID.</p>
```
[the remark this answers is not in the document]
Comment (sherenewso2) [reply en/docs/setup/configuring-a-read-only-ldap-user-store.md]: Fixed 6936454be19be2913232edb31b764ffa6080b416

File: en/docs/setup/configuring-a-jdbc-user-store.md [hunk 4]
```diff
@@ -178,7 +178,7 @@ For example :
 
 ``` toml
 [user_store]
-scim_enabled = true
+read_groups = true
```
Comment (IsuraD) [inline en/docs/setup/configuring-a-jdbc-user-store.md]: Why did we change this to read_groups?

**Pull request merged**